quantum/pointing_device_drivers.c

```diff
@@ -0,0 +1,262 @@
+/* Copyright 2017 Joshua Broekhuijsen <snipeye+qmk@gmail.com>
```

Comment:
Ideally these implementations would be in the driver code directly. Worth cleaning up later.

Reply:
Yeah, I had thought about that, and considered it, but wanted to stick with a more standard reporting for each sensor, as some support buttons and some don't.   And I didn't really want to introduce a new report just for the sensors.

Reply:
also, I planned on looking at adding support to the joystick stuff, later on.  And will probably clean it up more before then.